components/CredentialDetailsViews.vue

```diff
@@ -167,11 +167,28 @@ export default {
       backgroundColor: 'gray',
     };
 
+    // Fetch style, overrides, & highlights before component mounts
+    onBeforeMount(() => {
+      getDisplaysFromRenderMethod();
+    });
+
+    // Extract and parse images from credential's render method property
+    async function getDisplaysFromRenderMethod() {
+      if(props.credential?.renderMethod?.length) {
+        props.credential.renderMethod.forEach(rm => {
+          if(supportedRenderMethods.includes(rm.type)) {
+            credentialImages.push(rm.id);
```

Comment:
The `id` field is likely _not_ the long term home for URLs of the SVG (or other) files, so we should be ready to handle those variations when more render methods show up. Not something needed now, but something to ponder for the future.